<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutSettings</class>
 <widget class="QWidget" name="LayoutSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_left" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="CommonVideo" name="video" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_channel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
    outline: 0px;
    border: 0px;
    border-radius: 0px;
    padding-left: 0px;
    padding-right: 0px;
    min-width: 40px;
	min-height: 25px;
    color: #4A4A4A;
    background-color: #E1E1E1;
}
QPushButton:hover
{
    background-color: #d0d0d0;
}
QPushButton:pressed
{
    background-color: #2B9CC6;
}
QPushButton:checked
{
    background-color: #2B9CC6;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_channel"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_right" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widgetScreen" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelScreen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Screen Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="comboBoxScreen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_layoutButton" native="true">
        <property name="styleSheet">
         <string notr="true">QToolButton
{
border: 1px solid transparent;
border-radius: 3px;
padding-left: 5px;
padding-right: 5px;
}
QToolButton:hover
{
border: 1px solid #71D6FC;
}
QToolButton:checked
{
border: 1px solid #0AA9E3;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout1">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout1.png</normaloff>:/layout/layout/layout1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout4">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout4.png</normaloff>:/layout/layout/layout4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout8">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout8.png</normaloff>:/layout/layout/layout8.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout8_1">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout8_1.png</normaloff>:/layout/layout/layout8_1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout9">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout9.png</normaloff>:/layout/layout/layout9.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout12">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout12.png</normaloff>:/layout/layout/layout12.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout12_1">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout12_1.png</normaloff>:/layout/layout/layout12_1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout14">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout14.png</normaloff>:/layout/layout/layout14.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout16">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout16.png</normaloff>:/layout/layout/layout16.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout25">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout25.png</normaloff>:/layout/layout/layout25.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout32">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout32.png</normaloff>:/layout/layout/layout32.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout32_2">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout32_2.png</normaloff>:/layout/layout/layout32_2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_layout64">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/layout64.png</normaloff>:/layout/layout/layout64.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomLayoutButton" name="toolButtonCustom">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/resource.qrc">
             <normaloff>:/layout/layout/custom_layout_setting.png</normaloff>:/layout/layout/custom_layout_setting.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_preview" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_page">
        <property name="text">
         <string>Page: 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="style" stdset="0">
         <string>normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_currentPage" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_currentPage">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_apply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_back">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CommonVideo</class>
   <extends>QWidget</extends>
   <header>commonvideo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLayoutButton</class>
   <extends>QToolButton</extends>
   <header>CustomLayoutButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
